modules/ui/src/app/interceptors/error.interceptor.ts

```diff
@@ -57,17 +58,18 @@ export class ErrorInterceptor implements HttpInterceptor {
       catchError((error: HttpErrorResponse | TimeoutError) => {
         if (error instanceof TimeoutError) {
           this.notificationService.notify(
-            'Back End is not responding. Please, try again later.'
+            'Testrun is not responding. Please try again in a moment.'
           );
         } else {
           if (error.status === 0) {
             this.notificationService.notify(
-              'Back End is not responding. Please, try again later.'
+              'Testrun is not responding. Please try again in a moment.'
             );
           } else {
             this.notificationService.notify(
-              error.error?.error || error.message
+              `Something went wrong. Check the logs for details here ${LOGS_FOLDER}`
```

Comment:
I'm just querying with Aksinnia whether we include the full logs url as 1) it can vary if the user installs differently and 2) The logs should be viewed in the console windows for the current runtime.